File: Java/Program/Clothing.java
public class Clothing extends Product {
    private String size, material;
    private char gender;

    public Clothing(){
        this.size = "";
        this.material = "";
        this.gender = '\0';
    }

    public Clothing(String size, String material, char gender){
        this.size = size;
        this.material = material;
        this.gender = gender;
    }

    public void setSize(String size) {
        this.size = size;
    }

    public void setMaterial(String material) {
        this.material = material;
    }

    public void setGender(char gender) {
        this.gender = gender;
    }

    public String getSize() {
        return size;
    }

    public String getMaterial() {
        return material;
    }

    public char getGender() {
        return gender;
    }
}

File: Python/program/Clothing.py
from Product import Product

class Clothing(Product):

    __size = ""
    __material = ""
    __gender = ''

    def __init__(self, size = "", material = "", gender = ''):
        self.__size = ""
        self.__material = ""
        self.__gender = ""
    
    def getSize(self):
        return self.__size
    
    def getMaterial(self):
        return self.__material
    
    def getGender(self):
        return self.__gender
    
    def setSize(self, size):
        self.__size = size
    
    def setMaterial(self, material):
        self.__material = material
    
    def setGender(self, gender):
        self.__gender = gender